packages/flutter_tools/lib/src/build_system/build_system.dart

```diff
@@ -481,41 +480,59 @@ class _BuildInstance {
     final Stopwatch stopwatch = Stopwatch()..start();
     bool passed = true;
     bool skipped = false;
-    try {
-      final bool canSkip = await node.computeChanges(environment, fileCache);
+
+    // The build system should produce a list of aggregate input and output
+    // files for the overall build. The goal is to provide this to a hosting
```

Comment:
This list is provided to a hosting...

or

This list can then be consumed by a hosting...

Reply:
Done